django-stubs/contrib/postgres/fields/ranges.pyi

```diff
@@ -21,32 +25,41 @@ class RangeOperators:
     NOT_GT: Literal["&<"]
     ADJACENT_TO: Literal["-|-"]
 
-class RangeField(models.Field):
+_Range = TypeVar("_Range", bound=Range[Any])
```

Comment:
nitpick: when using a name that is really close to an existing one `Range` vs `_Range`, I would prefer to have `Range` and `_RangeT`

Reply:
Oh, that seems more reasonable. Thanks! https://github.com/typeddjango/django-stubs/pull/2078/commits/b4c339cc37d8e1a5255005e1a3c92ab7460edc42